packages/office-ui-fabric-react/src/components/Facepile/examples/Facepile.Basic.Example.tsx

```diff
@@ -43,7 +43,7 @@ export class FacepileBasicExample extends React.Component<{}, IFacepileBasicExam
         <Facepile {...facepileProps} />
         <div className="control">
           <Slider
-            label="Number of Personas:"
```

Comment:
Hmmm strange that the Facepile example is being changed - do you know why?

Reply:
yeah so basically cause the new format for slider display is "Label: value" and the components that use slider for their example had some of the labels that already had a colon so basically just got rid of the extra colon from the examples

Reply:
I know we talked about this yesterday, but I forgot to say that maybe we should write that in the overview or best practices sections of the `Slider` page so that people know that it formats it this way.